pkg/consts/consts.go

```diff
@@ -0,0 +1,8 @@
+package consts
+
+const (
+	LabelKeyCreatedByOctopus = "testing.kyma-project.io/created-by-octopus"
+	LabelKeySuiteName        = "testing.kyma-project.io/suite-name"
+	LabelKeyTestDefName      = "testing.kyma-project.io/def-name"
+	TestingPodGeneratedName  = "octopus-testing-pod-"
```

Comment:
How about prefixing pod names with suite name? Just like Deployments do.

Reply:
but if u will use the Deployments then u need to remember to truncate it

Reply:
what is wrong with the proposed approach?